tests/installation/first_boot.pm

```diff
@@ -10,6 +10,12 @@ sub run() {
         return;
     }
 
+    # Wait for desktop to load after booting from an installed disk image
+    if ( check_var( "BOOT_HDD_IMAGE", 1 ) ) {
+        assert_screen "generic-desktop", 120;
+        return;
```

Comment:
yeah, that's due to somehow the desktop widget disappears, and it took long time no action then get into screensaver...it would be the same result even running on first_boot.pm in case the desktop widget really somehow disappears :(